src/Lithnet.GoogleApps/GmailServiceRequestFactory.cs

```diff
@@ -78,22 +89,52 @@ public IEnumerable<string> GetDelegates(string id)
 
         public IEnumerable<string> GetSendAsAddresses(string id)
         {
-            id.ThrowIfNotEmailAddress();
 
-            GmailService service = this.GetService(id);
-            UsersResource.SettingsResource.SendAsResource.ListRequest request = service.Users.Settings.SendAs.List(id);
-            ListSendAsResponse result = request.ExecuteWithRetry(RetryEvents.BackoffOAuthNotFound, this.RetryCount);
-            return result.SendAs?.Select(t => t.SendAsEmail) ?? new List<string>();
+            try
+            {
+                id.ThrowIfNotEmailAddress();
+
+                GmailService service = this.GetService(id, DefaultTimeout);
+                UsersResource.SettingsResource.SendAsResource.ListRequest request = service.Users.Settings.SendAs.List(id);
+                ListSendAsResponse result = request.ExecuteWithRetry(RetryEvents.Backoff | RetryEvents.Timeout, this.RetryCount);
```

Comment:
This was previously Backoff | OAuth | NotFound. Any reason for removing OAuth and NotFound? These can be fired off when a short time has passed between creating a user and calling these APIs

Reply:
Probably just a mistake on my side. Fixing (adding the previously deleted RetryEvents to the call).